src/main/java/santiagoAndFerdy/vgs/gridScheduler/GridSchedulerProxy.java

```diff
@@ -0,0 +1,79 @@
+package santiagoAndFerdy.vgs.gridScheduler;
```

Comment:
In the gs branch I split this up into `IGridSchedulerResourceManagerClient` and `IGridSchedulerGridSchedulerClient`.